components/Cta/Cta.styles.ts

```diff
@@ -7,9 +7,9 @@ import {
   type CtaIconLeftMarginType,
 } from './Cta.types';
 
-export const cta = 'group hocus:underline transition-all';
+export const cta = 'group/cta hocus:underline transition-all';
```

Comment:
Where there are multiple elements in a component with the `group` TW class sometimes it can cause conflicts. This ensure that the cta related group classes are for cta only